### PR TITLE
feat: compute_snapshot_schedule_create/delete/get/edit

### PR DESCRIPTION
**Pull request by gryczj**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by iennae** [comment]
@gemini review

File: compute/test/snapshotSchedule.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        cwd,
+      }
+    );
+
+    assert(
+      response.includes(`Snapshot schedule: ${snapshotScheduleName} created.`)
+    );
+  });
+
+  it('should delete snapshot schedule', () => {
```
[the remark this answers is not in the document]
Comment (iennae) [reply compute/test/snapshotSchedule.test.js:45]: isn't there a create snapshot schedule right before this? can you explain more what you are suggesting here?

[the remark this answers is not in the document]
Comment (gryczj) [reply compute/test/snapshotSchedule.test.js:45]: but this solution is already in the code. create-snapshot-test is called before delete-snapshot-test

[the remark this answers is not in the document]
Comment (gryczj) [reply compute/test/snapshotSchedule.test.js:54]: @iennae fixed

**Review by m-strzelczyk — APPROVED** [review]

File: compute/snapshotSchedule/createSnapshotSchedule.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        description: snapshotScheduleDescription,
+        snapshotSchedulePolicy:
+          new compute.ResourcePolicyInstanceSchedulePolicySchedule({
+            retentionPolicy:
+              new compute.ResourcePolicySnapshotSchedulePolicyRetentionPolicy({
+                maxRetentionDays: 5,
+              }),
+            schedule: new compute.ResourcePolicySnapshotSchedulePolicySchedule({
+              // Similarly, you can create a weekly or monthly schedule.
+              // Review the resourcePolicies.insert method for details specific to setting a weekly or monthly schedule.
```
[the remark this answers is not in the document]
Comment (gryczj) [reply compute/snapshotSchedule/createSnapshotSchedule.js:60]: @m-strzelczyk link to the docuemntation added

**Pull request merged**